Jenkinsfile

```diff
@@ -0,0 +1,36 @@
+#!groovy
+
+node {
+  checkout scm
+
+  def dockerRepoName = 'zooniverse/shakespeares_world'
+  def dockerImageName = "${dockerRepoName}:${env.BUILD_ID}"
+  def newImage = null
+
+  stage('Build Docker image') {
+    newImage = docker.build(dockerImageName)
+    newImage.inside {
+      sh 'npm install'
+    }
+  }
+
+  stage('Deploy') {
+    if (BRANCH_NAME == 'alpha') {
+      newImage.inside {
+        sh 'export BUCKET="zooniverse-static"; export PREFIX="preview.zooniverse.org/folgerdemo/"; export BASE_URL="https://preview.zooniverse.org/folgerdemo"; npm run build && npm run deploy'
+      }
+    }
+
+    // if (BRANCH_NAME == 'staging') {
```

Comment:
Is this supposed to be commented out? I'd guess this either needs to be uncommented, or we should remove it if it's not needed.